programs/nodeos/main.cpp

```diff
@@ -80,6 +82,7 @@ enum return_codes {
 
 int main(int argc, char** argv)
 {
+   eosio::chain::eosvmoc::start_compile_monitor();
```

Comment:
This is causing an undefined symbol on OSX.

Reply:
Right, this is just slop added in to get it work. See item #3 in review comments